src/Validation/Validator.php

```diff
@@ -189,6 +196,18 @@ public function __construct()
         $this->_providers = self::$_defaultProviders;
     }
 
+    /**
+     * Whether to apply last flag to rule(s).
+     * The first failing one will abort then per rule.
+     *
+     * @param bool $last If to apply last flag.
+     * @return void
+     */
+    public function enableLast(bool $last = true): void
```

Comment:
Would `stopOnFailure` or `stopOnError` be a better name? I know we used last in 3.x but I've never found that name to be overly clear.